ui/src/app/routing/guards.tsx

```diff
@@ -13,7 +13,7 @@ export const signInGuard: Guard = {
 };
 
 const userIsEnabled = (userDisabledInDb: boolean) =>
-    (!userDisabledInDb && eligibleForRegisteredTier(profileStore.get().profile.tierEligibilities));
+  (!userDisabledInDb && eligibleForTier(profileStore.get().profile, AccessTierShortNames.Registered));
```

Comment:
I just found out about  hasRegisteredTierAccess in access-tiers.tsx. Maybe we can have a ticket later to use just one

Reply:
This is slightly different.  It checks whether a user **can** have access vs. whether a user **does** have access.